src/PowerShellEditorServices.Engine/Services/Analysis/AnalysisService.cs

```diff
@@ -51,6 +54,11 @@ public class AnalysisService : IDisposable
 
         private static readonly string[] s_emptyGetRuleResult = new string[0];
 
+        private Dictionary<string, Dictionary<string, MarkerCorrection>> codeActionsPerFile =
```

Comment:
My current opinion on this is that fields are best set in the constructor, since it's explicit when the set is evaluated (and in what order)